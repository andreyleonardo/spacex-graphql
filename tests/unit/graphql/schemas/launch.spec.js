const glue = require('schemaglue');
const { schema } = glue('src/graphql/launch');

describe('Launch Schema', () => {
  it('schema matches with snapshot', () => {
    expect(schema).toMatchSnapshot();
  });
});
