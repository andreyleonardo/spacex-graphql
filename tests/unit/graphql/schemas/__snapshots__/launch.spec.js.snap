// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Launch Schema schema matches with snapshot 1`] = `
"type Launch {
  flight_number: Int
  mission_name: String
  launch_year: String
  launch_date_local: String
  launch_success: Boolean
  rocket: Rocket
}
type Query {

  launches: [Launch]
  launch(flight_number: Int!): Launch
}"
`;
